spec/controllers/enrollments_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe EnrollmentsController, type: :controller do
+  let(:white_player) { FactoryGirl.create(:player, email: 'blah@blah.com', password: 'SPACECAT') }
+  let(:black_player) { FactoryGirl.create(:player, email: 'meow@meow.com', password: 'MONORAILCAT') }
+  let(:game) { FactoryGirl.create(:game, white_player_id: white_player.id, black_player_id: nill) }
```

Comment:
I think you've got a typo here -- "nil" is just spelled with one "l." Also, I'm not sure if you saw this PR, but when I input the feature that allowed the player to choose their own color, the id left nil was actually forced into 0 (since it's an integer value). So I might change this to 0 to reflect that circumstance.